minvws/nl-kat-coordination
# Also delete self-affirming or self-infered objects

### Changes

Also delete self affirming objects

### Issue link

Relevant to but not resolving #3432 (see also #3499)

### Demo
![image](https://github.com/user-attachments/assets/6adffb1f-3423-4462-9bba-2e95f91639d4)
Cannot exist anymore

### QA notes
Use https://github.com/originalsouth/VisualOctopoes to see that the graph provided image get garbage collected
Test for any unusual regression in particular for KATFindings.

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
